### ISSUE TITLE
sculpt boot stuck on splash screen on T490s, suspecting bender

### ISSUE THREAD

**Issue by jschlatow** [genodelabs/genode#4186]
With this issue, I want to share and document my experiences with the boot issue on my thinkpad t490s.

When I first tried booting Sculpt 21.03 on my t490s, it got stuck very early in the boot procedure. Unfortunately, the laptop does not have AMT and therefore I had no serial available for introspection. Booting in legacy mode instead of UEFI (to get vga console output) was not successful either as the laptop's firmware has a problem with booting from storages with GPT and bios boot partition. After a couple of iterations, I found out that removing bender from the grub.cfg "solved" the issue.

Today I had a further look at this issue to find a better solution. I tried booting via PXE to get some console output from bender. Interestingly, in this setup, I am able to boot sculpt with bender. By iterating the different firmware options, I discovered that the boot procedure only gets stuck in bender if the UEFI/legacy startup mode is set to "Both". This is reproducible with the t490.

In summary, the following sculpt boot configurations work:
- with bender:
  - PXE boot in legacy mode with
  - booting from USB stick or NVMe in UEFI-only mode
- without bender:
  - booting from USB stick or NVMe in "Both" mode with UEFI-first
